lib/fs/basicfs_copy_range_ioctl.go

```diff
@@ -56,38 +36,28 @@ func copyRangeIoctl(src, dst basicFile, srcOffset, dstOffset, size int64) error
 
 	if srcOffset == 0 && dstOffset == 0 && size == 0 {
 		// Optimization for whole file copies.
-		var errNo syscall.Errno
 		_, err := withFileDescriptors(src, dst, func(srcFd, dstFd uintptr) (int, error) {
-			_, _, errNo = syscall.Syscall(syscall.SYS_IOCTL, dstFd, FICLONE, srcFd)
-			return 0, nil
+			return 0, unix.IoctlFileClone(int(dstFd), int(srcFd))
 		})
 		// Failure in withFileDescriptors
 		if err != nil {
 			return err
 		}
```

Comment:
No need for this if anymore, just return err straight through

Reply:
Good point, done.